nxviz/plots.py

```diff
@@ -695,7 +707,8 @@ def draw_nodes(self):
                         rotation=rot,
                         rotation_mode="anchor",
                         color=self.node_label_color[i],
-                        fontsize=10,
+                        fontsize=self.fontsize,
+                        family=self.fontfamily
```

Comment:
I'd like to suggest renaming this keyword argument to `fontfamily`, rather than `family`, to keep it explicit that this refers to a font property.

```suggestion
                        fontfamily=self.fontfamily
```

Reply:
> I'd like to suggest renaming this keyword argument to `fontfamily`, rather than `family`, to keep it explicit that this refers to a font property.
> 
> Suggested change Beta 
> [Give us feedback](https://www.research.net/r/SuggestedChanges)
>                         family=self.fontfamily
>                         fontfamily=self.fontfamily

The self.ax.text() takes "family" as a key argument and thus cannot be changed

Reply:
Ah! Thank you for the clarification. I misunderstood, because I didn't see that the kwarg belonged to `ax.text()`. Thanks for the clarification!